Dockerfile

```diff
@@ -1,12 +1,19 @@
 FROM python:3.8-slim
-RUN pip install --no-cache-dir pipenv
+
+RUN useradd -d /app -m appuser
+USER appuser
```

Comment:
Kubernetes with a pod security policy runAsNonRoot unfortunatly can't validate if user is non-root if it isn't numeric.
distroless for instance sets 65532. https://github.com/GoogleContainerTools/distroless/blob/main/base/base.bzl

You would need to specify runAsUser so it works.

Reply:
As an alternative bitnami's python image most likely will have non-root preconfigured.

Reply:
I'm not too fond of their image, and it also doesn't cover the other containers that don't need a Python environment, such as hbbtv_stream_check_api.
Also, I don't quite understand what the issue is. Do you mean that the "USER appuser" statement will stay as-is in the resulting container manifest? I can hard-code a numeric uid if this helps.

Reply:
And to justify _why_ I don't like the bitnami images: The way they're building their minideb base image is very obscure and hackish. I'd prefer that we don't use something that is hard to understand. I also don't see how they implemented their rootless setup - it's certainly not done in the Dockerfile: https://github.com/bitnami/bitnami-docker-python/blob/3.9.5-prod-debian-10-r47/3.9-prod/debian-10/Dockerfile

Reply:
It just needs to be numeric

Reply:
https://github.com/kubernetes/kubernetes/issues/59819

Reply:
The USER flag is now numeric (uid 65532).